cmd2/cmd2.py

```diff
@@ -2979,9 +2979,17 @@ def run(filename: str):
                     error_msg = "Error opening script file '{}': {}".format(expanded_filename, ex)
                     self.perror(error_msg, traceback_war=False)
 
+            # noinspection PyShadowingBuiltins
+            def quit():
+                """Function callable from the interactive Python console to exit that environment"""
+                raise EmbeddedConsoleExit
+
+            # Set up Python environment
             bridge = PyscriptBridge(self)
-            self.pystate['run'] = run
             self.pystate[self.pyscript_name] = bridge
+            self.pystate['run'] = run
+            self.pystate['quit'] = quit
```

Comment:
Looks good.  Would it be reasonable to add a couple new unit tests to cover the added functionality?

Reply:
If it isn't easy to add a unit test, perhaps add an example?